src/graphql/queries/properties.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from '@apollo/client';
+
+export const GET_PROPERTIES = gql`
+  query Properties {
+    recentProperties {
+      area
+      bathrooms
+      bedrooms
+      address {
+        city
+        state
+        district
+        streetNumber
+      }
+      imageUrls
+      buyPrices {
+        total
+      }
+    }
+  }
+`;
+
+export interface Address {
+  city: string;
+  state: string;
+  district: string;
+  streetNumber: string;
+}
+
+export interface BuyPrices {
+  total: number;
+}
+
+export interface Property {
+  area: string;
+  bathrooms: number;
+  bedrooms: number;
+  address: Address;
+  imageUrls: string[];
+  buyPrices: BuyPrices;
+}
+
+export interface QueryData {
```

Comment:
por padrao, a gente usa o nome da query para nomear isso

```suggestion
export interface RecentPropertiesResponse {
```

o que acha?

Reply:
ficou ótimo